vms/platformvm/block/builder/helpers_test.go

```diff
@@ -362,61 +358,81 @@ func defaultFx(t *testing.T, clk *mockable.Clock, log logging.Logger, isBootstra
 	return res
 }
 
-func buildGenesisTest(t *testing.T, ctx *snow.Context) []byte {
+func buildGenesisTest(t *testing.T, ctx *snow.Context) *genesis.Genesis {
 	require := require.New(t)
 
-	genesisUTXOs := make([]api.UTXO, len(preFundedKeys))
+	genesisUTXOs := make([]*genesis.UTXO, len(preFundedKeys))
 	for i, key := range preFundedKeys {
-		id := key.PublicKey().Address()
-		addr, err := address.FormatBech32(constants.UnitTestHRP, id.Bytes())
-		require.NoError(err)
-		genesisUTXOs[i] = api.UTXO{
-			Amount:  json.Uint64(defaultBalance),
-			Address: addr,
+		addr := key.PublicKey().Address()
+		genesisUTXOs[i] = &genesis.UTXO{
+			UTXO: avax.UTXO{
+				UTXOID: avax.UTXOID{
+					TxID:        ids.Empty,
+					OutputIndex: uint32(i),
+				},
+				Asset: avax.Asset{ID: ctx.AVAXAssetID},
+				Out: &secp256k1fx.TransferOutput{
+					Amt: defaultBalance,
+					OutputOwners: secp256k1fx.OutputOwners{
+						Locktime:  0,
+						Threshold: 1,
+						Addrs:     []ids.ShortID{addr},
+					},
+				},
+			},
+			Message: nil,
 		}
 	}
 
-	genesisValidators := make([]api.GenesisPermissionlessValidator, len(genesisNodeIDs))
-	for i, nodeID := range genesisNodeIDs {
-		addr, err := address.FormatBech32(constants.UnitTestHRP, nodeID.Bytes())
-		require.NoError(err)
-		genesisValidators[i] = api.GenesisPermissionlessValidator{
-			GenesisValidator: api.GenesisValidator{
-				StartTime: json.Uint64(defaultValidateStartTime.Unix()),
-				EndTime:   json.Uint64(defaultValidateEndTime.Unix()),
-				NodeID:    nodeID,
+	vdrs := txheap.NewByEndTime()
+	for idx, nodeID := range genesisNodeIDs {
```

Comment:
We initialize `genesisNodeIDs` in `init()`, so I think it would make sense for us to also initialize them here as well. Maybe we can move that blob of code down here?

Reply:
But then what if we write a test that does not really use the environment? The genesisNodeIDs content would change depending on the order tests are executed right?
This is the reason I'd keep them in the init,